script/DeployUSDC.s.sol

```diff
@@ -85,23 +76,57 @@ contract DeployUSDCRinkeby is DeployUSDCScript {
         address admin = owner;
         IERC20Metadata usdc = new TestERC20('USD Coin', 'USDC');
 
-        IOUToken iou = new IOUToken('Nouns USDC IOU', 'NOUUSDC', DECIMALS, msg.sender);
+        Payer payer = new Payer(owner, usdc);
 
-        Payer payer = new Payer(owner, usdc, iou);
+        PriceFeed priceFeed = new PriceFeed(
+            AggregatorV3Interface(RINKEBY_USDC_ETH_CHAINLINK),
+            USDC_ETH_CHAINLINK_HEARTBEAT,
+            PRICE_LOWER_BOUND,
+            PRICE_UPPER_BOUND
+        );
 
-        iou.grantRole(iou.MINTER_ROLE(), address(payer));
-        iou.grantRole(iou.BURNER_ROLE(), address(payer));
+        new TokenBuyer(
+            usdc,
+            priceFeed,
+            USD_POSITION_IN_USD * 10**DECIMALS, // baselinePaymentTokenAmount
+            0, // minAdminBaselinePaymentTokenAmount
+            2 * USD_POSITION_IN_USD * 10**DECIMALS, // maxAdminBaselinePaymentTokenAmount
+            0, // botIncentiveBPs
```

Comment:
nit: reminder to rename these inline comments to `botDiscountBPs` etc.

Reply:
fixed